xtest/tdfs.py

```diff
@@ -100,6 +100,10 @@ def __init__(self, **kwargs: dict[str, Any]):
         if self.semver >= (0, 4, 28):
             self.features.add("connectrpc")
 
+        # public_client_id removed in service 0.5.6
+        if self.semver >= (0, 5, 6):
+            self.features.add("no-public-client-id")
```

Comment:
had to reverse the logic here bc main is still 0.5.5 because it hasnt been released yet but public_client_id has been removed